docs/order-confirmation/index.md

```diff
@@ -157,21 +194,20 @@ Error: This expression has type React.element array
        but an expression was expected of type React.element
 ```
 
-We get this error primarily because collections in OCaml can only contain
-elements of the same type. The `tbody` element expects children of type
-`React.element`[^3], but the call to `Js.Array.map` returns `array(React.element)`,
-which creates a type mismatch. To make the actual type match the expected type,
-we must add a call to `React.array` which turns `array(React.element)` to
-`React.element`.
+We get this error because the `tbody` element expects children of type
```

Comment:
This is minor, but there's certain repetition between the "we'd get this error" in line 187 and the one in this line. Maybe change with

```suggestion
The compiler warns us because the `tbody` element expects children of type
```
? or st similar

Reply:
Rephrased in https://github.com/melange-re/melange-for-react-devs/pull/10/commits/f580d453198ae43a1718ead1b480af340c30bcbe